modules/ui/entity_issues.js

```diff
@@ -117,12 +117,6 @@ export function uiEntityIssues(context) {
             .merge(itemsEnter)
             .classed('expanded', function(d, i) { return i === _expanded; });
 
-
-        items.select('button.label')     // propagate bound data
-            .attr('title', function(d) {
-                return t('issues.' + d.severity + 's.icon_tooltip');
-            });
-
```

Comment:
FYI the `title` tooltip was originally just on the icon, to help correlate the icon with the severity title. Not sure how it ended up on the whole label. We probably don't need it anyway.

Reply:
Nice - that's what I assumed.. It makes some sense on the icon part only, but not really necessary since these icons aren't for clicking.

Reply:
I was vaguely thinking that the icon could be the "zoom-to-issue" button too. This would make more sense once we show the badges on the map, since the icons would be the same. But even in this case we wouldn't use the same tooltip text.